### PR TITLE
Feature/bowling citizen animations

### PR DESCRIPTION
**Pull request by Matt9440**
Tied in ShadowBrains bowling animations into the game; some logic will need replacing once we have more of the game logic implemented.

Press R to cycle between rotating your character (to aim) and positioning your character.

### CODE REVIEW

File: code/Gamemodes/Bowling/Entities/BowlingBallCarriable.cs [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 
-public class BowlingBallCarriable : BaseCarriable
+public partial class BowlingBallCarriable : BaseCarriable
 {
 	public BowlingBall BowlingBall { get; set; }
-
-	TimeSince timeSinceLastThrow = 0;
+	[Net]
+	public bool HasThrown { get; set; }
+	[Net, Predicted]
```
Comment (DevulTj) [inline code/Gamemodes/Bowling/Entities/BowlingBallCarriable.cs:8]: I'd newline before the next net prop

File: code/Gamemodes/Bowling/Entities/BowlingBallCarriable.cs [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+	TimeSince TimeSinceLastThrow { get; set; }
+	BowlingPlayer BowlingPlayer => Owner as BowlingPlayer;
```
Comment (DevulTj) [inline code/Gamemodes/Bowling/Entities/BowlingBallCarriable.cs:10]: And here too imo

File: code/Gamemodes/Bowling/Player/BowlingPlayer.cs [hunk 3]
```diff
@@ -1,15 +1,54 @@
...
+﻿using Sandbox;
 
-public class BowlingPlayer : BasePlayer
+namespace Sports;
+
+public partial class BowlingPlayer : BasePlayer
 {
+	[Net]
+	public bool HasThrown { get; set; }
+	public BowlingPlayerAnimator PlayerAnimator => GetActiveAnimator() as BowlingPlayerAnimator;
```
Comment (DevulTj) [inline code/Gamemodes/Bowling/Player/BowlingPlayer.cs:9]: And here

**Review by DevulTj — CHANGES_REQUESTED** [review]

File: code/Gamemodes/Bowling/Entities/BowlingBallCarriable.cs (hunk 2 above)
Comment (Adesii) [reply code/Gamemodes/Bowling/Entities/BowlingBallCarriable.cs:13]: this probably should also have accesors.
so private public or anything

**Review by Adesii — APPROVED** [review]
Looks good to me

**Pull request merged**